src/appmixer/utils/ai/GenerateImage/GenerateImage.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
         if (imageUrl) {
             const response = await context.httpRequest.get(imageUrl, { responseType: 'stream' });
             const readStream = response.data;
-            const filename = `generated-image-${new Date}.png`;
+            const filename = `generated-image-${(new Date).toISOString()}.png`;
             const file = await context.saveFileStream(filename, readStream);
             return context.sendJson({ fileId: file.fileId, prompt, size }, 'out');
```

Comment:
- wouldn't be a good idea to send also the image url to the output? 
 - `if (imageUrl) { ` is there a change there is no `image` in the response? shouldn't we throw an error in such case?

Reply:
I wouldn't send imageUrl since that resides OpenAI while these utilities are opaque when it comes to the service behind them.